### DISCUSSION TITLE
求助，华为的harmony系统如何使用手机端蓝灯app?

### DISCUSSION THREAD

**Discussion by WizardJason** [getlantern/lantern-forum-cn#521] [Q&A]
手机是华为的harmony系统，下载安卓6.0+的app，安装后打不开无法正常运行,求解决方案

**Comment by MaxMKMax** [comment]
@WizardJason 您好. 华为暂时用不了蓝灯. 请问您可以试看用电脑吗?

**Reply by WizardJason** [reply]
但是我之前一直都用的了，突然更新了之后就不行了

**Reply by WizardJason** [reply]
有没有老版本的app呢  我试试 @Mike777Khoo

**Comment by MaxMKMax** [comment]
@WizardJason 您好. 我们没有老版的app了. 但您可以试看重新下载.
https://github.com/getlantern/lantern

https://lantern.io/zh-Hans/faq
如果还遇到什么问题, 您可以参考上面的链接. 谢谢您.
